src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,45 @@
+package duke;
+
+import duke.task.Task;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Ui {
+
+    static final String divider = "____________________________________________________________";
```

Comment:
Please follow the naming convention for static final variables. Constant names must be all uppercase using underscore to separate words. [See here](https://se-education.org/guides/conventions/java/basic.html#naming)